src/mux-analytics.brs

```diff
@@ -970,6 +974,66 @@ function muxAnalytics() as Object
     m._addEventToQueue(m._createEvent("playbackmodechange", props))
   end sub
 
+  prototype.requestHandler = sub(message as Object)
+    requestVariant = message.request_variant
+
+    props = {}
+    props.request_start = message.request_start
+    props.request_hostname = message.request_hostname
```

Comment:
we probably, sadly, need to make sure none of these are `invalid`, or catch/handle `invalid` later down the pipe

Reply:
@daytime-em, I've added the changes you mentioned:
- `type` is now required. If it's missing, the event is ignored.
- I implemented throughput and latency based on the JS implementation and set them in `dynamicProperties` so they're sent with each beacon.
  - I added a check to only use numeric values: if a string comes in, a warning is logged and it's ignored for the calculation. I did this because parsing was causing issues and was truncating very large Unix timestamp values.

I'd appreciate it if you could review it and try it out to see if it looks correct.

Thanks!